action/protocol/staking/vote_reviser.go

```diff
@@ -47,6 +48,12 @@ func (vr *VoteReviser) Revise(ctx protocol.FeatureCtx, csm CandidateStateManager
 		case err != nil:
 			return err
 		}
+		if vr.shouldCorrectCandSelfStake(height) {
```

Comment:
`shouldClearUnstakedSelfStake` is a better name, also should this be put after L57 `shouldReviseSelfStakeBuckets()`?